src/xdsl/dialects/std.py

```diff
@@ -41,3 +43,25 @@ class Return(Operation):
     @staticmethod
     def get(*ops: Union[Operation, SSAValue]) -> Return:
         return Return.build(operands=[[op for op in ops]])
+
+
+@irdl_op_definition
+class Branch(Operation):
+    name: str = "std.br"
```

Comment:
I am uncertain how close we want to be with MLIR here, but AFAIK, MLIR has the `cf` dialect now for control flow instructions. Maybe add another dialect to this? Or don't and make sure to handle it in the MLIRConverter properly.

Reply:
I moved them to the cf dialect.